topology/probes/netlink.go

```diff
@@ -373,11 +373,14 @@ Loop:
 	for {
```

Comment:
The code should be :    (package "sync/atomic")

	for ;atomic.LoadInt32(&u.running) > 0 ; {
		n, err := syscall.EpollWait(epfd, events[:], 1000)
		if err != nil {
			errno, ok := err.(syscall.Errno)
			if ok && errno != syscall.EINTR {
				logging.GetLogger().Error("Failed to receive from netlink messages: %s", err.Error())
			}
			continue
		}
if n == 0 {
      continue
}
		msgs, err := s.Receive()